pinot-core/src/test/java/org/apache/pinot/queries/InnerSegmentAggregationSingleValueQueriesTest.java

```diff
@@ -49,27 +49,49 @@
       " GROUP BY column1, column3, column6, column7, column9, column11, column12, column17, column18";
 
   @Test
-  public void testAggregationOnly() {
-    String query = "SELECT" + AGGREGATION + " FROM testTable";
```

Comment:
For aggregation tests with FILTER clause, can we please also have some tests where filter is on a transform function and not just identifier ? Let's also ensure to have all combinations tested - mix of aggregations with and without FILTER and with/without outer predicate

Reply:
Why are we modifying an existing test case. Is the existing test case no longer valid? If not, I would suggest leaving the existing test case as is and adding a new test case.